mathjax/mathjax.php

```diff
@@ -16,12 +16,19 @@ function mathjax_install() {
     Addon::registerHook('page_header', 'addon/mathjax/mathjax.php', 'mathjax_page_header');
     Addon::registerHook('addon_settings', 'addon/mathjax/mathjax.php', 'mathjax_settings');
     Addon::registerHook('addon_settings_post', 'addon/mathjax/mathjax.php', 'mathjax_settings_post');
+    Addon::registerHook('template_vars','addon/mathjax/mathjax.php', 'mathjax_template_vars');
     logger('installed js_math addon');
 }
+
+function mathjax_template_vars($a, &$arr) {
+    $arr['vars']['addon_hooks'][] = "mathjax";
```

Comment:
What if the `addon_hooks` key doesn't exist in the array? Wouldn't it trigger a Notice message?

Reply:
apparently it doesn't, at least in my environment (php 7.0)
I enabled `E_NOTICE` and had many NOTICE messages, but not in `addon/mathjax/mathjax.php`.

If you think I should add some defensive code anyway, I'll be happy to add it.